c/cert/test/rules/ERR30-C/test.c

```diff
@@ -0,0 +1,72 @@
+#include <errno.h>
+#include <limits.h>
+#include <stdio.h>
+#include <stdlib.h>
+
```

Comment:
I think since setlocale is known to be bad, we should probably have a test case for it?

Reply:
@mbaluda -- since it MIGHT set `errno` it seems like the most concervative thing would be to just warn for cases where errno is used after calling this function.